.infrastructure/deployment.yml

```diff
@@ -0,0 +1,52 @@
+apiVersion: apps/v1
+kind: Deployment
+
+metadata:
+  name: todoapp
+  namespace: mateapp
```

Comment:
Ensure that the `namespace` specified (`mateapp`) exists in your Kubernetes cluster. If it doesn't, you need to create it or change this value to a valid namespace.